src/o365/graph/commands/teams/teams-app-install.ts

```diff
@@ -0,0 +1,125 @@
+import * as request from 'request-promise-native';
+import auth from '../../GraphAuth';
+import Utils from '../../../../Utils';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import { CommandOption, CommandValidate } from '../../../../Command';
+import GraphCommand from '../../GraphCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  appId: string;
+  teamId: string;
+}
+
+class GraphTeamsAppInstallCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.TEAMS_APP_INSTALL}`;
+  }
+
+  public get description(): string {
+    return 'Install an app from the catalog to a Microsoft Team';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const endpoint: string = `${auth.service.resource}/v1.0/`
+
+    auth
+      .ensureAccessToken(auth.service.resource, cmd, this.debug)
+      .then((): request.RequestPromise => {
+
+        const requestOptions: any = {
+          url: `${endpoint}/teams/${args.options.teamId}/installedApps`,
```

Comment:
This will give you an invalid URL with `//`, eg. `https://graph.microsoft.com/v1.0//teams...`